src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/r/RCompletionToolTip.java

```diff
@@ -54,13 +57,55 @@ public void resolvePositionAndShow(String signature,
             rectangle.getTop());
    }
    
+   public void resolvePositionAndShow(String signature)
+   {
+      setCursorAnchor();
+      resolvePositionAndShow(signature, docDisplay_.getCursorBounds());
+   }
+   
+   private String truncateSignature(String signature)
```

Comment:
It looks to me like we truncate at > 200 rather than > 500 (which I think is what the PR summary said).  What's your intention? It seems to me like truncating at somewhere around 200-300 would be fine.